cmd/nerdctl/run.go

```diff
@@ -274,6 +274,8 @@ func setCreateFlags(cmd *cobra.Command) {
 	})
 	cmd.Flags().String("cosign-key", "", "Path to the public key file, KMS, URI or Kubernetes Secret for --verify=cosign")
 	// #endregion
+
+	cmd.Flags().String("isolation", "default", "Specify isolation technology for container. On Linux valid value is default. On Windows can also be host or process")
```

Comment:
Should we add that on Windows this defaults to process isolation?

Reply:
updated